dubbo-remoting/dubbo-remoting-api/src/main/java/org/apache/dubbo/remoting/api/MultiplexProtocolConnectionManager.java

```diff
@@ -41,10 +40,16 @@ public Connection connect(URL url) {
     }
 
     @Override
-    public void forEachConnection(Consumer<Connection> connectionConsumer) {
-        protocols.values().forEach(p -> p.forEachConnection(connectionConsumer));
+    public ConnectionPool getConnectionPool(URL url) {
```

Comment:
`ConnectionPool` is similiar to `ConnectionManager` , can these merged into one ?